src/components/user/tour/create-tour/index.js

```diff
@@ -0,0 +1,162 @@
+import React, {Component} from 'react';
```

Comment:
Filename and Component name must be the same. 
There is no need to have an `index.js` file.